scripts/prototype-slow-consumer.py

```
@@ -0,0 +1,294 @@
+def summarise(label: str, values: list[float], frames: int) -> None:
+    if not values:
+        print(f"  {label:22s} no frames rendered ({frames} total)")
+        return
+    values = sorted(values)
+    p95 = values[min(len(values) - 1, int(0.95 * len(values)))]
+    print(f"  {label:22s} n={len(values):3d}  p50 {statistics.median(values):7.1f} ms  "
+          f"p95 {p95:7.1f} ms  max {values[-1]:7.1f} ms")
+
+
+async def main() -> None:
+    api = subprocess.Popen(
```

Ruff (0.16.1)

[warning] Async functions should not create subprocesses with blocking methods

(ASYNC220)